src/redux/categories/categorySlice.js

```diff
@@ -0,0 +1,18 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Kindly don't disable your linters and try to fix your linters manually 👍 because linters prevents your from introducing those errors which can break your app 😨